### PR TITLE
Issue #153. Fix the config false statement in last-error

### PR DESCRIPTION
**Pull request by mjethanandani**
Issue #153

### CODE REVIEW

File: draft/draft-ietf-idr-bgp-model.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="US-ASCII"?>
 <!DOCTYPE rfc SYSTEM "rfc2629.dtd">
-<rfc category="std" docName="draft-ietf-idr-bgp-model-latest" ipr="trust200902">
+<rfc category="std" docName="draft-ietf-idr-bgp-model-latest" ipr="trust200902" consensus="true">
```
Comment (jhaas-pfrc) [inline draft/draft-ietf-idr-bgp-model.xml:3]: I suggest that changes to the code outside the scope of the issue get fixed under their own issue.

File: src/validate-and-gen-trees.sh [hunk 2]
```diff
@@ -98,7 +98,7 @@ rm ../bin/*-rib-tree.txt.tmp
 
 echo "Validating examples"
 
-for i in yang/example-bgp-configuration-*.xml
+for i in yang/example-bgp-configuration-a.*.xml
```
Comment (jhaas-pfrc) [inline src/validate-and-gen-trees.sh:101]: I suggest that changes to the code outside the scope of the issue get fixed under their own issue.

Comment (mjethanandani) [reply src/validate-and-gen-trees.sh:101]: I would love to, and understand the sentiment behind keeping the commit limited to the immediate issue at hand.

Except without this change, I am getting compile errors and the draft does not build and to fix all of that requires me to do even more surgery than I would like to.

File: draft/draft-ietf-idr-bgp-model.xml (hunk 1 above)
Comment (mjethanandani) [reply draft/draft-ietf-idr-bgp-model.xml:3]: I can remove this.

File: src/validate-and-gen-trees.sh (hunk 2 above)
Comment (jhaas-pfrc) [reply src/validate-and-gen-trees.sh:101]: Understood.  It wasn't an obvious place where the change would result in errors.  I did try it out and don't think I noticed anything.  I must have missed it.

**Review by jhaas-pfrc — APPROVED** [review]

**Pull request merged**